policyengine_us/parameters/gov/states/az/tax/income/exemptions/parents_grandparents.yaml

```diff
@@ -0,0 +1,14 @@
+description: Arizona provides an exemption of this amount per qualifying parents or grandparents.
```

Comment:
```suggestion
description: Arizona provides an exemption of this amount per qualifying parent or grandparent.
```

Reply:
one exemption for each (f.e. mother and father one each)?

Reply:
yes, the legal code says: a resident taxpayer is allowed an exemption of $10,000 for each parent or ancestor of a parent of the taxpayer, who is sixty-five years of age or older.